includes/panopto_admin.form.inc

```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+ * Defines the admin settings form.
+ *
+ * @param array $form
+ *   The Drupal form definition.
+ * @param array $form_state
+ *   The Drupal form state.
+ *
+ * @return array
+ *   The Drupal form definition.
+ */
+function islandora_video_streaming_panopto_admin(array $form, array &$form_state) {
+  $form['islandora_video_streaming_panopto_host'] = array(
```

Comment:
Please add user-facing instructions in this form describing where in Panopto to find/generate the client id, client secret, and folder id.  Also note the requirement that this uses a "User Based Server Application" as opposed to other API Client types.